libraries/hotstuff/CMakeLists.txt

```diff
@@ -15,3 +15,10 @@ target_include_directories( hotstuff
                           )
 
 add_subdirectory( test )
+
+install(TARGETS hotstuff
```

Comment:
A directory inside `library/chain` makes sense to me. Do you want that done as part of this PR?

Reply:
Is this too much ask for you for now? Maybe it is best to do now, as it would make things simpler and avoid big changes later.

Reply:
Before we do this, I think we should get @arhag take on it.

Reply:
I agree that this would be better, and probably better to do this in this PR because of the CMakeLists.txt changes. I think we can still have separate classes for `qc_chain` and `pacemaker`, and have these insulated by not accessing the controller directly, but as they are an integral part of the chain behavior, I really feel they should be in `libraries/chain`. 
I don't think they need a separate directory, I don't see the benefit.

Reply:
The benefit of a diff directory to me is it keeps the hotstuff consensus logic separate.

Reply:
My personal thinking is that hotstuff It is just in two classes, and I don't think there will be ambiguity where it is implemented. I feel that the separate directory is unnecessary. Why not have separate dirs for the fork database, the abi_serializer, etc...?

Reply:
Yes, we should get Areg's opinion. You can have your current PR in as is.